ethcore/account-state/Cargo.toml

```diff
@@ -1,28 +1,37 @@
 [package]
-description = "Ethereum accounts, keeps track of changes to the code and storage."
-name = "state-account"
+description = "Ethereum accounts, keeps track of changes to the code and storage"
+name = "account-state"
 version = "0.1.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
 common-types = { path = "../types"}
+derive_more = "0.15.0"
 ethereum-types = "0.6.0"
 ethtrie = { package = "patricia-trie-ethereum", path = "../../util/patricia-trie-ethereum" }
+evm = { path = "../evm" }
```

Comment:
In future it would be good to clean up some of the dependencies. I don't think the `Ethereum accounts` should require `evm` dependency.

Reply:
We use `CallType` from the `evm`. Maybe we can move that to `common_types`.